### PR TITLE
use RSpec.current_example for RSpec 3.x

### PR DESCRIPTION
**Pull request by kurotaky**
@mizoR Hello. 😃 
`RSpec::Core::ExampleGroup#example` is deprecated, so I remove `example` and I added `RSpec.current_example`. Thank you.

The following error message appears.

``` sh
# Run `bundle exec rspec`.
F

Failures:

  1) EmailDoc::Document#render should render document
     Failure/Error: %r<## #{example.description}>,
       `example` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
     # ./spec/lib/email_doc/document_spec.rb:27:in `block (3 levels) in <top (required)>'

Finished in 0.00075 seconds (files took 0.27196 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/lib/email_doc/document_spec.rb:24 # EmailDoc::Document#render should render document
```

### CODE REVIEW

**Comment by kurotaky** [comment]
Sorry, I added other bugs...

File: spec/helper.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 require 'rspec'
 require 'email_doc'
+require 'active_support/rescuable'
```
Comment (mizoR) [inline spec/helper.rb:3]: Just question: Why is this change required? 🤔

Comment (kurotaky) [reply spec/helper.rb:3]: Because of rspec commands failed. :(

RSpec commands caused failed messages as follows:

```
F

Failures:

  1) EmailDoc::Document#render should render document
     Failure/Error:
       Class.new(ActionMailer::Base) do
         def welcome(email)
           mail \
             to:       email,
             from:     'from@example.org',
             reply_to: 'noreply@example.org',
             subject:  'WELCOME',
             body:     'Welcome to my site.'
         end
       end

     NameError:
       uninitialized constant ActiveSupport::Rescuable
     # ./spec/lib/email_doc/document_spec.rb:5:in `require'
     # ./spec/lib/email_doc/document_spec.rb:5:in `block (2 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:19:in `block (2 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:24:in `block (3 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:37:in `block (4 levels) in <top (required)>'
     # ./spec/lib/email_doc/document_spec.rb:37:in `each'
     # ./spec/lib/email_doc/document_spec.rb:37:in `block (3 levels) in <top (required)>'

Finished in 0.37025 seconds (files took 0.27632 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/lib/email_doc/document_spec.rb:26 # EmailDoc::Document#render should render document
```

**Comment by kurotaky** [comment]
@mizoR 
I fixed codes for RSpec3.x. Please check it.😃

File: spec/helper.rb (hunk 1 above)
Comment (mizoR) [reply spec/helper.rb:3]: Aha, but I actually feel that it is the responsibility of action_mailer...🤔  (I mean that [action_mailer/rescuable.rb](https://github.com/rails/rails/blob/master/actionmailer/lib/action_mailer/rescuable.rb#L6) should require [active_support/rescuable.rb](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/rescuable.rb) ??

Anyway, this change is OK for me, thank you 👍

File: lib/email_doc/document.rb [hunk 2]
```diff
@@ -5,22 +5,22 @@ module EmailDoc
   class Document
     extend Forwardable
 
-    attr_reader :mail, :context
+    attr_reader :mail, :context, :ex
```
Comment (mizoR) [inline lib/email_doc/document.rb:8]: nits: I don't think that "ex" is good naming.  Because most of programmers understand "ex" is an instance of Exception (not Example).

But it is nits issue, so I will merge this PR now~!

**Comment by mizoR** [comment]
Released this as [v0.0.2](https://rubygems.org/gems/email_doc/versions/0.0.2).  Thank you for your good work 😄

File: lib/email_doc/document.rb (hunk 2 above)
Comment (kurotaky) [reply lib/email_doc/document.rb:8]: `example` is not available from within an `it` block. so I use `ex` in render methods.
However, as you say, `ex` is It would associate multiple meanings(e.g. `Exception`, `Example`, `Exercise`). 😕  I'll think about it in the next chance!

**Comment by kurotaky** [comment]
Thanks!
